README.md

```diff
@@ -1,5 +1,7 @@
 # Adyen [online payment](https://docs.adyen.com/online-payments) integration demos
 
+Checkout sample application using Adyen Drop-in v6 (check [folder /v5](./v5) to access the previous version using Adyen Drop-in v5).
```

Comment:
nit: instead of `check` -> `see`
Add relative link. For V5, let's put this in an archive_folder:
_archive
|
--- /v5

We should always show latest in the root folder